Project-I-Social-Card/social-card/src/components/CardComponents/CardContainer.js

```diff
@@ -1,2 +1,15 @@
 import React from 'react';
 import './Card.css';
+
+import CardContent from './CardContent.js'
+
+
+const Card = () => {
```

Comment:
This seems a bit unnecessary to have a component that just returns another component without passing any props